packages/ui-kit/package.json

```diff
@@ -37,6 +37,8 @@
     "@graphql-codegen/typescript-operations": "^4.0.1",
     "@graphql-codegen/typescript-react-query": "^6.0.0",
     "@juggle/resize-observer": "^3.4.0",
+    "@mui/base": "^5.0.0-beta.31",
+    "@mui/utils": "^5.15.4",
```

Comment:
Those should be in `dependencies` instead of `devDependencies`, right?

Reply:
Well, it depends. I made it work in `devDependencies` since it's been used as a foundation for our own component rather than just re-export. As building blocks, we may keep it in dev.

Reply:
In theory if a dependency is not in `dependencies` it is not added into the bundle, therefore the hook and components we are using from base-ui would return `undefined`, I tested adding Chart.js to dev dependencies and it works as well though it shouldn't.

I think I would keep those in `dependencies` just for safety in case those would break in a production environment, wdyt?

Reply:
Yeah, I also think this should be a normal dependency.

Reply:
It actually works as `dependencies` because of a lib-build-specific behavior. Basically, we’re talking about not running an app but rather building a lib. When we build a library with rollup (or similar bundlers like webpack), the bundler includes all imported modules in the final bundle unless instructed otherwise. It doesn’t differentiate between `dependencies` and `devDependencies`. If a module is imported into your code, it is considered part of the bundle.